static/app/views/projectInstall/issueAlertOptions.spec.tsx

```diff
@@ -31,126 +20,25 @@ describe('IssueAlertOptions', function () {
     setProvider: jest.fn(),
   };
 
-  const props = {
-    onChange: jest.fn(),
-    organization,
-    notificationProps,
-  };
-  const getComponent = () => <IssueAlertOptions {...props} {...notificationProps} />;
```

Comment:
Before this was doing something equiv to `<IssueAlertOptions org={org} notificationProps={notificationProps} {...notificationProps} />` which is all the things.